docs/aerodromes/classd/Parafield.md

```diff
@@ -8,9 +8,23 @@
 | Name               | Callsign       | Frequency        | Login ID                         |
 | ------------------ | -------------- | ---------------- | ---------------------------------------- |
 | **Parafield ADC**  | **Parafield Tower**  | **118.700**          | **PF_TWR**                        |
+| Parafield ADC 2†  | Parafield Tower  | 124.600        | PF-2_TWR                        |
```

Comment:
PF_TWR 118.700
PF_E_TWR 124.600
(See below)

Reply:
> PF_TWR 118.700 PF_E_TWR 124.600 (See below)

@Slep-wt this too

Reply:
> > PF_TWR 118.700 PF_E_TWR 124.600 (See below)
> 
> @Slep-wt this too

Currently we have it as PF-2_TWR as y'all know, if possible I'd like to keep it as such. In terms of denoting who owns what, editing the controller info in Positions.xml to state exactly what runways each position is responsible for would be a good way to go about it.